demonstrations/ahs_aquila.py

```diff
@@ -95,22 +95,22 @@
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 In this treatment of the Hamiltonian, we will assume that we are operating such that we only allow
-access to two states; the low and high energy states are referred to as the ground (:math:`\ket{g}`)
-and Rydberg (:math:`\ket{r}`) states respectively.
+access to two states; the low and high energy states are referred to as the ground (:math:`| g \rangle`)
+and Rydberg (:math:`| r \rangle`) states respectively.
 
 Qubit interaction in a Rydberg atom system is mediated by a mechanism called Rydberg blockade, which arises
 due to van der Waals forces between the atoms. This is described by the interaction Hamiltonian:
 
 .. math:: \hat{H}_{int} = \sum_{j=1}^{N-1}\sum_{k=j+1}^{N} \frac{C_6}{R^6_{jk}}\hat{n}_j\hat{n}_k
 
-where :math:`n_j=\ket{r_j}\bra{r_j}` is the number operator acting on atom :math:`j`, :math:`R_{jk} = \lvert x_j - x_k \lvert` is the
+where :math:`n_j=| r_j \rangle \langle r_j |` is the number operator acting on atom :math:`j`, :math:`R_{jk} = \lvert x_j - x_k \lvert` is the
 distance between atoms :math:`j` and :math:`k`, and :math:`C_6` is a fixed value determined by the nature of the ground
 and Rydberg states (for Aquila, :math:`5.24 \times 10^{-24} \text{rad m}^6 / \text{s}`, referring to the
 :math:`| 70S_{1/2} \rangle` state of the Rb-87 atom).
 
 There are two key things to be aware of in the interaction term. First, the energy contribution of
 the interaction between each pair of atoms is only non-zero when both atoms are in the Rydberg state,
-so that :math:`\bra{\psi}\hat{n}_k \hat{n}_j\ket{\psi}=1`. Second, the energy contribution for each
+so that :math:` \langle \psi |  \hat{n}_k \hat{n}_j | \psi \rangle  =1`. Second, the energy contribution for each
```

Comment:
```suggestion
so that :math:`\langle \psi | \hat{n}_k \hat{n}_j | \psi \rangle  =1`. Second, the energy contribution for each
```

This latex block doesn't render correctly.

Reply:
ah, thanks!